package.json

```diff
@@ -2,17 +2,15 @@
   "name" : "@project-ncl/pnc-api-types-ts",
   "version" : "3.0.4-pnc-ab29765",
   "devDependencies" : {
-    "dtsgenerator" : "^2.7.0",
-    "replace" : "^1.2.1"
+    "dtsgenerator" : "^3.19.2",
```

Comment:
@DnsZhou Hey! Did you try to run release.sh locally? I couldn't make it work after npm/node upgrade without upgrading also dependencies. While upgrading dts I found out they split it to plugins and feature we used (namespace replacement) is now plugin. This needs new dependency and is configured in dtsgen.json.

Reply:
Hi @jomrazek , I have only tried to use `mvn clean install -DskipTests` in `.tmp/pnc/rest-api ` of pnc-api-types-ts` after I ran `/release.sh 8ade3a9b527cebe0d7607dd8d98a1bd7cdac3e56`(which is `PNC 3.0.4 RC1`) because I want to figure out the blocker of the license header. it failed at first and then I manually copied `.license-header.txt` into the pnc/rest-api/ folder and the release was success.

my local environment：
 - Java: Open JDK 11.0.25
 - Maven: 3.8.8
 - Node: 22.13.0
 - npm:11.0.0

Reply:
Yeah that's because I'm building just rest-api module and that expected header is in parent, I talked about it with Dustin and at the end we need just openapi.json and not to build whole pnc, so I skipped license check to make it pass this step. Everything should be fixed now, be aware that if you don't want to release this library manually locally you need to first update node/npm on jenkins with Dustin to new required versions. Currently release job will still fail because of old npm/node.